specification/electronic-prescription-service-api.yaml

```diff
@@ -18,12 +18,11 @@ info:
     ### As a prescriber
     You can:
     * encode prescription data
-    * create a secondary care (outpatient) prescription
-    * cancel a secondary care (outpatient) prescription
+    * create and cancel a secondary care (outpatient) prescription
+    * create and cancel a primary care prescription
 
```

Comment:
Would these be better as individual bullets? e.g.
 * create a secondary care (outpatient) prescription
 * cancel a secondary care (outpatient) prescription
 * create a primary care prescription
 * cancel a primary care prescription